src/programs.py

```diff
@@ -0,0 +1,628 @@
+import os
+import sys
+from src.header import logger
+
+import src.utilities as util
+import src.header as header
+import src.runmodes
+from src.runcard_parsing import PROGRAMruncard
+from src.program_interface import ProgramInterface
+
```

Comment:
I'll merge master into this before continuing so that all src point already to pyhepgrid.src

Reply:
I wrote "I'll" when I wanted to say "I'd" :P